Tests/Unit/ExtensionDataSharingTests.m

```diff
@@ -72,10 +74,10 @@ - (void)testEnablingDataSharingFromMainApp {
     [Parse _resetDataSharingIdentifiers];
 
     _testHelper.runningInExtensionEnvironment = YES;
-    XCTAssertThrows([Parse enableDataSharingWithApplicationGroupIdentifier:@"yolo"]);
+    [Parse enableDataSharingWithApplicationGroupIdentifier:@"yolo"];
 
     // Just to make sure that initialization runs smoothly
-    [Parse setApplicationId:[[NSUUID UUID] UUIDString] clientKey:[[NSUUID UUID] UUIDString]];
+    XCTAssertThrows([Parse setApplicationId:[[NSUUID UUID] UUIDString] clientKey:[[NSUUID UUID] UUIDString]]);
```

Comment:
This probably will change back if we move the validation to `ParseClientConfiguration` itself.